examples/opencv_rc.py

```diff
@@ -0,0 +1,632 @@
+        :return: None
+        """
+
+        # Stop the display
+        self._display.stop()
+
+        # Stop the controller
+        self._ctrl.stop()
+
+        # Disconnect from Cozmo
+        self._cozmo_clt.disconnect()
+        self._cozmo_clt.stop()
+
+
+class RemoteControl(object):
+    """
+    Use the keyboard to control Cozmo's movements, as well as OpenCV's track bars to set the robot's head angle, lift
+    height, and both its linear and angular velocities.
+    """
+
+    def __init__(self, clt, win_name='Control Panel'):
+        """
+        Initialize the different variables required to remote control Cozmo using both keyboard events and OpenCV's
+        taskbars.
+        :param clt: pycozmo.Client. An instance of the client used to communicate with Cozmo.
+        :param win_name: String. A string representing the title of the window used to display OpenCV's track bars.
+        """
+
+        # Declare a flag telling the main thread when to stop its execution
+        self.go_on = True
+
+        # Set the client used for communicating with Cozmo
+        self._cozmo_clt = clt
+
+        # Keep track of Cozmo's current action
+        self._action = {'linear': Direction.NONE, 'angular': Direction.NONE}
+
+        # Keep track of Cozmo's linear and angular velocities
+        self._velocity = {'linear': 0, 'angular': 0}
+
+        # Keep track of the head light state
+        self._head_light = False
+
+        # Keep track of the head tilt angle
+        self._head_tilt = pc.MIN_HEAD_ANGLE.radians
+
+        # Keep track of the lift's height
+        self._lift_height = pc.MIN_LIFT_HEIGHT.mm
+
+        # Initialize a listener to monitor keyboard events
+        self._kbd_listener = kbd.Listener(on_press=self._on_keypress, on_release=self._on_keyrelease)
+
+        # Other miscellaneous parameters to keep track of
+        self._win_name = win_name
+
+    def init(self):
+        """
+        Create the window OpenCV will use to display the different track bars, as well as start the keyboard listener.
+        :return: None
+        """
+
+        # Create a new thread for the window containing the task bars to prevent any lag issues due to key events
+        cv.startWindowThread()
+        # Create a window for the control panel
+        cv.namedWindow(self._win_name)
+
+        # Create the different trackbars controlling the robot's velocities,
+        # head tilt, and lift height
+        cv.createTrackbar('Linear velocity', self._win_name, 0, 100, self._on_linear_velocity_change)
+        cv.createTrackbar('Angular velocity', self._win_name, 0, 100, self._on_angular_velocity_change)
+        cv.createTrackbar('Head tilt', self._win_name, 0, 100, self._on_head_tilt_change)
+        cv.createTrackbar('Lift height', self._win_name, 0, 100, self._on_lift_height_change)
+        cv.createTrackbar('Head light', self._win_name, 0, 1, self._on_head_light_change)
```

Comment:
All the display code should be in the Display class.
You can pass references to the controller's change callbacks.

Reply:
I put that code here since in my mind it was not really part of the display per se. However, it can easily be moved in the Display class as you said. I will have a look at that later today.

Reply:
As you suggested I have moved the window and track bars creation into the display class. The RemoteController's callbacks are passed as argument to the `Display.init()` function. It is not pretty but it works, and it is more flexible that way.

Reply:
> I put that code here since in my mind it was not really part of the display per see.

But this is what creates the display window and the sliders in it, without this you don't have a display, right? :) I think it's better to have it separate from the controls, what if you'd like to use another interface for the video streaming instead of opencv?